packages/kotti-ui/source/kotti-field-select/KtFieldSingleSelect.vue

```diff
@@ -25,6 +25,16 @@
 					:label="option.label"
 					:value="option.value"
 				/>
+				<div v-if="actions" class="kt-field-select-actions">
+					<div
+						v-for="action in actions"
+						:key="action.value"
```

Comment:
this seems pretty unintentional after going through your code, you can for a fixed string  + `index`